orm/utils.go

```diff
@@ -181,18 +181,36 @@ func ToInt64(value interface{}) (d int64) {
 	return
 }
 
-// snake string, XxYy to xx_yy
+// snake string, XxYy to xx_yy , XxYY to xx_yy
```

Comment:
but our rules is all the capital will change to `_char`, I admire that would make more sense. but that change the behaves maybe will broken the exist database.